e2e-tests/cypress/tests/integration/channels/markdown/markdown_text_spec.js

```diff
@@ -49,7 +49,7 @@ describe('Markdown message', () => {
     testCases.forEach((testCase) => {
         it(testCase.name, () => {
             // #  Post markdown message
-            cy.postMessageFromFile(`markdown/${testCase.fileKey}.md`).wait(TIMEOUTS.FIVE_SEC);
+            cy.postMessageFromFile(`markdown/${testCase.fileKey}.md`).wait(TIMEOUTS.TWO_SEC);
```

Comment:
Other 3 failed.

Reply:
@saturninoabril 
I need to make be ask Dev to fix this. In those 3 cases, When mark down text is copied there's `\n` coming up.
and not <br>. I'll check with devs once.